extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -34,13 +34,29 @@ export class AmpMustache extends AMP.BaseTemplate {
   /** @override */
   compileCallback() {
     /** @private @const {string} */
-    this.template_ = this.element./*OK*/innerHTML;
+    const shadowTemplate = this.element.cloneNode(true);
+    this.nestedTemplates_ = {};
+    let index = 0;
+    shadowTemplate.content.querySelectorAll('template').forEach(
```

Comment:
Danger! `NodeList.prototype.forEach` will break in IE11. Use [`src/dom.js#iterateCursor`](https://github.com/ampproject/amphtml/blob/master/src/dom.js#L621) instead.